.pylintrc

```diff
@@ -518,7 +518,7 @@ max-branches=12
 max-locals=15
 
 # Maximum number of parents for a class (see R0901).
-max-parents=7
+max-parents=10
```

Comment:
Can you please help justify this change?

Reply:
there are new command-line options added to the tool. This increase is to accommodate those command-line argument parsing. For parents its required as the arguments get passes to parent class